### PR TITLE
Add some code comment and remove unused loop for dataset_reader

### PR DESCRIPTION
**Pull request by pengshuang**
See the closed issue https://github.com/allenai/allennlp-bert-qa-wrapper/issues/18
1. I updated some code comments in case that someone may be misunderstood.
2. I removed unused loop logic to make the code clear.

### CODE REVIEW

File: pretrained_bert/dataset_reader.py [hunk 1]
```diff
@@ -140,6 +140,7 @@ def is_whitespace(char):
                 break
             start_offset += min(length, self._document_stride)
 
+        #
```
Comment (pdasigi) [inline pretrained_bert/dataset_reader.py]: Empty comment?

File: pretrained_bert/dataset_reader.py [hunk 2]
```diff
@@ -140,6 +140,7 @@ def is_whitespace(char):
...
         for (doc_span_index, doc_span) in enumerate(doc_spans):
```
Comment (pdasigi) [inline pretrained_bert/dataset_reader.py]: Would you mind removing this for loop altogether and just take `doc_spans[0]` to make it even clearer what's happening here?

**Review by pdasigi — APPROVED** [review]
Thanks for doing this! Generally looks good. Would be great if you want to also make the minor change below.

File: pretrained_bert/dataset_reader.py (hunk 1 above)
Comment (pengshuang) [reply pretrained_bert/dataset_reader.py]: oh, I forgot to delete the empty comment and I have fixed.

File: pretrained_bert/dataset_reader.py (hunk 2 above)
Comment (pengshuang) [reply pretrained_bert/dataset_reader.py]: I have your idea and I will work on it.

Comment (pengshuang) [reply pretrained_bert/dataset_reader.py]: Need I create a new pull request to add this part ? (removing this for loop altogether...)

Comment (pengshuang) [reply pretrained_bert/dataset_reader.py]: I have removed unused loop logic to make the code clear. Feel free to contact me if you have any suggestions.

**Review by pdasigi — APPROVED** [review]
Thanks for removing the loop. LGTM.

**Pull request merged**